Sources/Services/ContainerSandboxService/SandboxService.swift

```diff
@@ -748,6 +762,15 @@ public actor SandboxService {
     ) {
         czConfig.process.arguments = [process.executable] + process.arguments
         czConfig.process.environmentVariables = process.environment
+
+        if czConfig.sockets.contains(where: {
```

Comment:
This will automatically set an environment variable even if the user specify `-ssh`, in the case where they've mounted a socket at `/run/host-services/ssh-auth.sock`, and they've not set the variable explicitly?

That's probably harmless but it might be unexpected. Using `if config.ssh && config.sockets.contains...` would ensure that we only do this when `--ssh` is specified. If `--ssh` isn't present, then nothing ever happens.

Reply:
can the `config.ssh` test move out of the closure to L768?